src/components/Table/Table.jsx

```diff
@@ -27,15 +28,17 @@ import {
   Avatar,
   VStack,
   HStack,
+  Spacer,
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Spacer' is defined but never used.